config/application.yml

```diff
@@ -0,0 +1,11 @@
+# Add configuration values here, as shown below.
```

Comment:
You accidentally pushed this! You should be using development.env and test.env for your environment variables